tests/test_logind.py

```diff
@@ -118,6 +122,13 @@ def test_session(self):
         self.assertRegex(out, 'Id=c1')
         self.assertRegex(out, 'LockedHint=yes')
 
+        session_mock.Lock()
+        self.assertRegex(self.p_mock.stdout.read(),
+                         b'emit org.freedesktop.login1.Session.Lock\n')
```

Comment:
This test isn't very interesting, as it does not check any behaviour through the official loginctl CLI; it just checks its own mock code. It doesn't hurt, of course, but that part is already well covered.